packages/feedback-service/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "feedback",
```

Comment:
It should be feedback

Reply:
yes i can see that is feedback only ? please elaborate your comment.

Reply:
@anjneeksharma he meant the `feedback` should be `Feedback` notice the caps